seed/seed.json

```diff
@@ -3037,6 +3037,37 @@
                 ]
             },
             "name": "NewiOSPlaylistUIStudy"
+        },
+        {
+            "experiments": [
+                {
+                    "feature_association": {
+                        "enable_feature": [
+                            "SkipUndecryptablePasswords"
+                        ]
+                    },
+                    "name": "Enabled",
+                    "probability_weight": 100
+                },
+                {
+                    "name": "Default",
+                    "probability_weight": 0
+                }
+            ],
+            "filter": {
+                "min_version": "127.1.68.128",
```

Comment:
Thanks for raising the Griffin stuff! 👍🏻
We could also put the first Beta (`127.1.68.107` — https://github.com/brave/brave-browser/issues/33548#issuecomment-2274939576).

Reply:
Updated!